src/Keboola/DbExtractor/Extractor/Common.php

```diff
@@ -212,25 +214,26 @@ function ($table) {
                 'type' => (isset($table['TABLE_TYPE'])) ? $table['TABLE_TYPE'] : '',
                 'rowCount' => (isset($table['TABLE_ROWS'])) ? $table['TABLE_ROWS'] : '',
             ];
-            if ($table["AUTO_INCREMENT"]) {
-                $tableDefs[$table['TABLE_SCHEMA'] . '.' . $table['TABLE_NAME']]['autoIncrement'] = $table['AUTO_INCREMENT'];
+            if ($table['AUTO_INCREMENT']) {
```

Comment:
myslim ze asi bude nejlepsi definovat driv neco jako `$currentTableId = $table['TABLE_SCHEMA'] . '.' . $table['TABLE_NAME']` .  pak muzem delat assignment na jedno radku.  jak je to tady tedko je trosku tezky cist imho.

Reply:
you're right :+1: fixed